src/qnngds/devices/snspd.py

```diff
@@ -4,12 +4,14 @@
 
 import phidl.geometry as pg
 from typing import Tuple, Optional, Union
+from qnngds.utilities import PadPlacement, QnnDevice
+import qnngds.utilities as utility
 
 
 def basic(
```

Comment:
maybe we could change the default size for the SNSPDs so it's a little easier to tell what's going on in the docs, since they kinda just look like a big rectangle at the moment.